### ISSUE TITLE
[V3 Docs] User guides

### ISSUE THREAD

**Issue by laggron42** [Cog-Creators/Red-DiscordBot#1734]
The Red ReadTheDocs page currently has installation informations and API/coding guides. I think it would be good to also include some user guides, like "getting started" or detailed cog guides, as an alternative to the help command.

I can write these docs, I just need to know what you think about it and what would be the format. I already wrote detailed docs for my repo's cogs and I'm planning to do it the same way for core cogs. 

Please take a look at the cog guides section of my [ReadTheDocs page](https://laggrons-dumb-cogs.readthedocs.io) and see if it the format is ok for core cogs.

**Comment by mikeshardmind** [comment]
I feel like most of the commands in the bot do not need this level of explanation. However, I do think there are certain features in the bot which could be better documented. I think this is overall a good idea, but leave off the installation section since it should be covered by a page explaining how to load cogs, and I think it may make sense to only build this level of detailed documentation for cogs and commands which are not expected to be receiving modifications for a while to prevent it from becoming an extra step all the time (Not to never do it, but to wait on these until closer to release) (for instance, commands in the general cog are unlikely to change much currently, but things like mod and permissions are still getting improvements right now as needed)

**Comment by laggron42** [comment]
I can remove the installation section and just write a getting started guide, like on V2

**Comment by laundmo** [comment]
as someone looking into using red it would be nice to have at least a copy paste of the help messsage somewhere. almost every bot has a command list, and i think the standard cogs for this should too.

i dont want to have to install it just to figure out the exact commands it has. this wouldnt be a problem with a hosted public bot, but since this is selfhosted i would have to install just to test. i need to know if its worth it before that.

**Comment by palmtree5** [comment]
Here's the current status of this issue:

| Cog | Done | In Progress | Not Started | 
|---|---|---|---|
| Admin | x |   |   |
| Alias | x |   |   |
| Audio |   | x |   |
| Bank | x |   |   |
| Cleanup | x |  |  |
| CogManagerUI | x |  |  |
| Core | x |  |  |
| CustomCommands | x |  |  |
| Dev | x |  |  |
| Downloader | x |  |  |
| Economy | x |  |  |
| Filter | x |  |  |
| General | x |  |  |
| Image | x |  |  |
| Mod | x |  |  |
| ModLog | x |  |  |
| Mutes | x |  |  |
| Permissions | x |  |  |
| Reports | x |  |  |
| Streams | x |  |  |
| Trivia | x |  |  |
| Warnings | x |  |  |

Technically the permissions cog has its cog reference that goes over how the permissions work but it is not a command reference
